programs/merkle-tree/src/verifier_invoked_instructions/insert_two_leaves_transaction.rs

```diff
@@ -62,3 +61,48 @@ pub fn process_insert_two_leaves<'info, 'a>(
 
     Ok(())
 }
+
+// #[derive(Accounts)]
```

Comment:
would it make sense to leave a todo?

Reply:
this is obsolete moved to account compression